### ISSUE TITLE
Again GW OKEX Unable to read wallet data positions.

### ISSUE THREAD

**Issue by r8m** [ctubio/Krypto-trading-bot#825]
```
10/09 14:59:40.658945 DEBUG OKEX <<< {"error_code":10007,"result":false}.
10/09 14:59:40.659501 GW OKEX Unable to read wallet data positions.
```
Getting same problem with wallet data:
_Originally posted by @r8m in https://github.com/ctubio/Krypto-trading-bot/issues/536#issuecomment-428165492_

**Comment by serzhiio** [comment]
Try with new API/SECRET pair.

**Comment by r8m** [comment]
> Try with new API/SECRET pair.

had tried this and also made manual trades through K web ui and okex web but no luck.

**Comment by ctubio** [comment]
etz fix this at #828
